### DISCUSSION TITLE
mongoDB github login won't work

### DISCUSSION THREAD

**Discussion by michaelnakonechny** [neuefische/questions#204] [Web]
already set my mail to public, deleted cache etc...still get an error message. any solutions for this? or should i just try to login with my google account?

**Comment by MerleMak** [comment]
Hi Michael, please add a screenshot of the error message and a pull request of the current changes you made to your code, so we can have a look :)

**Reply by michaelnakonechny** [reply]
<img width="384" alt="Bildschirmfoto 2022-10-05 um 09 48 47" src="https://user-images.githubusercontent.com/109967687/194008252-5dcaa175-20d8-479a-a70d-3becfe550309.png">

**Reply by nikstaaar** [reply]
I had the same Problem and just used google to login.

**Comment by MerleMak** [comment]
Did Niklas suggestion work already?
